benchmark/worker/benchmark_client.go

```diff
@@ -349,8 +355,9 @@ func (bc *benchmarkClient) doCloseLoopStreaming(conns []*grpc.ClientConn, rpcCou
 // getStats returns the stats for benchmark client.
 // It resets lastResetTime and all histograms if argument reset is true.
 func (bc *benchmarkClient) getStats(reset bool) *testpb.ClientStats {
-	var timeElapsed float64
+	var timeElapsed, elapseUtime, elapseStime float64
```

Comment:
renamed `elapsedUtime` -> `uTimeElapsed` and `elapseStime` -> `sTimeElapsed` here and elsewhere.